src/bitcoind/mod.rs

```diff
@@ -113,13 +116,34 @@ fn check_bitcoind_network(
     Ok(())
 }
 
+fn check_bitcoind_version(bitcoind: &BitcoinD) -> Result<(), BitcoindError> {
+    let network_info = bitcoind.getnetworkinfo()?;
+    let bitcoind_version = network_info
+        .get("version")
+        .and_then(|v| v.as_u64())
+        .ok_or_else(|| {
+            BitcoindError::Custom("No valid 'version' in getnetworkinfo response?".to_owned())
+        })?;
```

Comment:
I think we should just panic in this case.
```suggestion
        .expect("No valid 'version' in getnetworkinfo response?");
```

Reply:
I saw a similar error handling in [here](https://github.com/revault/revaultd/blob/fd54a81111c2450530de4879de6318110a3869e7/src/bitcoind/mod.rs#L97) and assumed that bitcoind might not respond also for the case of propping for version if it is warming up. If this is not the case then we can safely panic.

Reply:
Heh, no. You are right. Let's keep it like that for now.